pkg/filesystem/mount_unix.go

```diff
@@ -0,0 +1,130 @@
+//go:build darwin || freebsd || linux
```

Comment:
I would call this file `mount_linux.go`, because `fsconfig()` is Linux specific. Other operating systems could have a `mount_unsupported.go` or something.